documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -37,28 +37,39 @@ When using Kafka with a Kafka cluster not managed by Strimzi, you can specify th
 [id='con-common-configuration-ssl-{context}']
 === `ssl`
 
-Use the three allowed `ssl` configuration options for client connection using a specific _cipher suite_ for a TLS version.
-A cipher suite combines algorithms for secure connection and data transfer.
+You can incorporate SSL configuration and cipher suite specifications to further secure TLS-based communication between your client application and a Kafka cluster.
+In addition to the standard TLS configuration, you can specify a supported TLS version and enable cipher suites in the configuration for the Kafka broker.
 
-You can also configure the `ssl.endpoint.identification.algorithm` property to enable or disable hostname verification.
+A cipher suite is a set of algorithms for secure connection and data transfer.
+For example, a cipher suite specification might use a combination of the following algorithms:
+
+* TLS authentication
+* ECDHE (Elliptic Curve Diffie-Hellman Ephemeral) key exchange
+* RSA authentication
+* AES (Advanced Encryption Standard) encryption
+* SHA384 (Secure Hash Algorithm) data integrity protection
+
+The combination is encapsulated in the `TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384` cipher suite specification.
+
+The `ssl.enabled.protocols` property specifies the available TLS versions that can be used for secure communication between the cluster and its clients. 
+The `ssl.protocol` property sets the default TLS version for all connections, and it must be chosen from the enabled protocols.
+Use the `ssl.endpoint.identification.algorithm` property to enable or disable hostname verification. 
 
 .Example SSL configuration
 [source,yaml,subs="attributes+"]
 ----
 # ...
 spec:
   config:
-    ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" <1>
-    ssl.enabled.protocols: "TLSv1.2" <2>
-    ssl.protocol: "TLSv1.2" <3>
-    ssl.endpoint.identification.algorithm: HTTPS <4>
+    ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" # <1>
+    ssl.enabled.protocols: "TLSv1.3", "TLSv1.2" # <2>
+    ssl.protocol: "TLSv1.3" # <3>
+    ssl.endpoint.identification.algorithm: HTTPS # <4>
 # ...
 ----
-<1> The cipher suite for TLS using a combination of `ECDHE` key exchange mechanism, `RSA` authentication algorithm,
-`AES` bulk encyption algorithm and `SHA384` MAC algorithm.
-<2> The SSl protocol `TLSv1.2` is enabled.
-<3> Specifies the `TLSv1.2` protocol to generate the SSL context.
-Allowed values are `TLSv1.1` and `TLSv1.2`.
+<1> Cipher suite specification enabled.
+<2> TLS versions supported.
+<3> Default TLS version is `TLSv1.3`. If a client only supports `TLSv1.2`, it can still connect and communicate with the broker using that version instead.
```

Comment:
This seems a bit confusing -> the example is using path `.spec.config` which would suggest it is for clients. But the way the sentence here is phrased suggests it should be a server (a server - i.e. a Kafka broker - would have the path as `.spec.kafka.config`). I think these options can be applied for both sides. So should we make it more generic?
* Rephrase this with something like _Default TLS version is `TLSv1.3`. But if the other side only supports `TLSv1.2`, it can still connect and communicate with it using that version instead._
* Remove the `spec:` part from the example

Reply:
Right. I've mentioned the vice versa aspect. I also thought it would be useful to mention that this config can be on either end.